sysdig on Proxmox - sysdig-probe fails to load

I'm trying to run sysdig on Proxmox, and the sysdig-probe fails to load. 
Diagnostics: 

```bash
root@proxmox04:/tmp# pveversion -v
proxmox-ve-2.6.32: 3.4-156 (running kernel: 2.6.32-39-pve)
pve-manager: 3.4-9 (running version: 3.4-9/4b51d87a)
pve-kernel-2.6.32-39-pve: 2.6.32-157
lvm2: 2.02.98-pve4
clvm: 2.02.98-pve4
corosync-pve: 1.4.7-1
openais-pve: 1.1.4-3
libqb0: 0.11.1-2
redhat-cluster-pve: 3.2.0-2
resource-agents-pve: 3.9.2-4
fence-agents-pve: 4.0.10-3
pve-cluster: 3.0-18
qemu-server: 3.4-6
pve-firmware: 1.1-4
libpve-common-perl: 3.0-24
libpve-access-control: 3.0-16
libpve-storage-perl: 3.0-33
pve-libspice-server1: 0.12.4-3
vncterm: 1.1-8
vzctl: 4.0-1pve6
vzprocps: 2.0.11-2
vzquota: 3.1-2
pve-qemu-kvm: 2.2-11
ksm-control-daemon: 1.1-1
glusterfs-client: 3.5.2-1
```

```bash
root@proxmox04:/tmp# uname -r
2.6.32-39-pve
```

```bash
root@proxmox04:/tmp# ls -l /lib/modules/$(uname -r)
total 2368
lrwxrwxrwx  1 root root     36 Aug 14 09:03 build -> /usr/src/linux-headers-2.6.32-39-pve
drwxr-xr-x 12 root root   4096 Jun  5 11:28 kernel
-rw-r--r--  1 root root 605998 May  8 10:40 modules.alias
-rw-r--r--  1 root root 580612 May  8 10:40 modules.alias.bin
-rw-r--r--  1 root root   5665 May  8 10:40 modules.builtin
-rw-r--r--  1 root root   7318 May  8 10:40 modules.builtin.bin
-rw-r--r--  1 root root 205635 May  8 10:40 modules.dep
-rw-r--r--  1 root root 298611 May  8 10:40 modules.dep.bin
-rw-r--r--  1 root root     80 May  8 10:40 modules.devname
-rw-r--r--  1 root root  77081 May  8 10:40 modules.order
-rw-r--r--  1 root root    131 May  8 10:40 modules.softdep
-rw-r--r--  1 root root 273187 May  8 10:40 modules.symbols
-rw-r--r--  1 root root 342206 May  8 10:40 modules.symbols.bin
drwxr-xr-x  2 root root   4096 Jun  5 11:28 zfs
```

```bash
root@proxmox04:/tmp# sysdig-probe-loader
* Unloading sysdig-probe, if present
Error: Module sysdig_probe is not currently loaded
* Running dkms autoinstall

Kernel preparation unnecessary for this kernel.  Skipping...

Building module:
cleaning build area....
make KERNELRELEASE=2.6.32-39-pve -C /lib/modules/2.6.32-39-pve/build M=/var/lib/dkms/sysdig/0.1.102/build....(bad exit status: 2)
Error! Bad return status for module build on kernel: 2.6.32-39-pve (x86_64)
Consult /var/lib/dkms/sysdig/0.1.102/build/make.log for more information.
* Trying to load a system sysdig-probe, if present
* Trying to load a dkms sysdig-probe, if present
* Trying to find precompiled sysdig-probe for 2.6.32-39-pve
Found kernel config at /boot/config-2.6.32-39-pve
* Trying to download precompiled module from https://s3.amazonaws.com/download.draios.com/stable/sysdig-probe-binaries/sysdig-probe-0.1.102-x86_64-2.6.32-39-pve-f9c010e053612af9f0cf3856537bef42.ko
Download failed, consider compiling your own sysdig-probe and loading it or getting in touch with the sysdig community
root@proxmox04:/tmp#
```

```bash
root@proxmox04:/tmp# cat /var/lib/dkms/sysdig/0.1.102/build/make.log
DKMS make.log for sysdig-0.1.102 for kernel 2.6.32-39-pve (x86_64)
Fri Aug 14 09:10:34 BST 2015
expr: syntax error
make: Entering directory `/usr/src/linux-headers-2.6.32-39-pve'
  LD      /var/lib/dkms/sysdig/0.1.102/build/built-in.o
  CC [M]  /var/lib/dkms/sysdig/0.1.102/build/main.o
/var/lib/dkms/sysdig/0.1.102/build/main.c: In function ‘ppm_ioctl’:
/var/lib/dkms/sysdig/0.1.102/build/main.c:782:3: error: implicit declaration of function ‘for_each_process’ [-Werror=implicit-function-declaration]
/var/lib/dkms/sysdig/0.1.102/build/main.c:782:23: error: expected ‘;’ before ‘{’ token
/var/lib/dkms/sysdig/0.1.102/build/main.c:751:27: warning: unused variable ‘t’ [-Wunused-variable]
cc1: some warnings being treated as errors
make[1]: *** [/var/lib/dkms/sysdig/0.1.102/build/main.o] Error 1
make: *** [_module_/var/lib/dkms/sysdig/0.1.102/build] Error 2
make: Leaving directory `/usr/src/linux-headers-2.6.32-39-pve'
root@proxmox04:/tmp#
```

I installed the kernel headers using `apt-get install pve-headers-2.6.32-39-pve` after `apt-get -y install linux-headers-$(uname -r)` failed (`Unable to locate package linux-headers-2.6.32-39-pve`)

What can I do to get this to work? Thanks!

--- a/driver/main.c
+++ b/driver/main.c
@@ -610,7 +610,11 @@ static long ppm_ioctl(struct file *filp, unsigned int cmd, unsigned long arg)
 #ifdef for_each_process_thread
 		for_each_process_thread(p, t) {
 #else
+#ifdef for_each_process_all
+		for_each_process_all(p) {
+#else
 		for_each_process(p) {
+#endif
 			t = p;
 			do {
 				task_lock(p);
@@ -633,8 +637,13 @@ static long ppm_ioctl(struct file *filp, unsigned int cmd, unsigned long arg)
 		}
 #else
 				task_unlock(p);
+#ifdef while_each_thread_all
+			} while_each_thread_all(p, t);
+		}
+#else
 			} while_each_thread(p, t);
 		}
+#endif
 #endif
 
 		rcu_read_unlock();

